utils/src/main/kotlin/LicenseFilenamePatterns.kt

```diff
@@ -35,23 +35,23 @@ object LicenseFilenamePatterns {
     )
 
     /**
-     * A list of globs that match default patent file names.
+     * A list of globs that match default patent file names. The patterns are supposed to be used case-insensitively.
```

Comment:
This should be squashed to the previous commit.

Reply:
Done.